Functions.Templates/Templates/SqlOutputBinding-Python/function.json

```diff
@@ -0,0 +1,28 @@
+{
+    "scriptFile": "__init__.py",
+    "bindings": [
+      {
+        "authLevel": "anonymous",
+        "name": "req",
+        "direction": "in",
+        "type": "httpTrigger",
+        "methods": [
+          "get",
+          "post"
```

Comment:
Remove POST (although we should really figure out what we want these samples to support...)